.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+Issues submitted here should pertain to bugs/features of our [docs site](https://docs.fastlane.tools/).
```

Comment:
Just noticed that I didn't know what "pertain" means. Learning every day ;)
(If I am not the only one, you could replace it with "concern")

Reply:
I don't know the exact meaning either, as I'm not a native english speaker. @taquitos, What do you think?

Reply:
_pertain_ means "apply to", as in `Issues submitted here should only apply to bugs/features of our...`

I think it makes sense to change that word to something non-native english speakers would have an easier time with 😃

Reply:
> Issues submitted here should represent bugs/features of our [docs site](https://docs.fastlane.tools).